COPY/etc/NetworkManager/dispatcher.d/99-fix-issue

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# bail unless ($1 = eth0 && $2 = up) || ($1 = eth0 && $2 = hostname)
+# dont call exit
+if [ "$1" != "eth0" -o "$2" != "up" -a "$2" != "hostname" ] ; then
+  :
+else
+  # After a network interface is loaded
+  # this updates the /etc/issue file to show the ip address
+  # and product name
+
+  # TODO: parse /var/www/miq/vmdb/config/locales/en.yml
+  # extract PRODUCT from en:product:name
+  PRODUCT='ManageIQ'
+  IP=`hostname -I | cut -d' ' -f1`
+
+  # replace \p with the product name
+  # replace \i with the ip address
+
+  sed /etc/issue.template -e "s/\\\\P/${PRODUCT}/g" -e "s/\\\\i/${IP}/g" > /etc/issue
+  sed /etc/issue.template -e "s/\\\\P/${PRODUCT}/g" -e "s/\\\\i/${IP}/g" > /etc/issue.net
```

Comment:
I wouldn't repeat the subst code, so maybe simply copy the file:
cp /etc/issue /etc/issue.net

Thanks.